### PR TITLE
Styling for categories and inventory

### PR DESCRIPTION
**Pull request by samanthalai0914**
- added styling for images and text on categories screen

- added Firebase call for category DisplayName
- styling for category titles
- added Touchable Opacity components w/ placeholder icon for individual items

### CODE REVIEW

File: screens/InventoryByCategoryScreen.js [hunk 1]
```diff
@@ -3,8 +3,11 @@ import {
   ScrollView,
   StyleSheet,
   Text,
-  View
+  View,
+  TouchableOpacity,
```
Comment (wayne-li2) [inline screens/InventoryByCategoryScreen.js]: Convention is -- these should be alphabetical.

File: screens/InventoryByCategoryScreen.js [hunk 2]
```diff
@@ -39,17 +44,37 @@ export default class InventoryByCategoryScreen extends React.Component {
           inventory: inventory
         });
       }.bind(this));
+
+    await firebase
```
Comment (wayne-li2) [inline screens/InventoryByCategoryScreen.js]: Let's not do this -- instead, let's pass the categoryDisplayName like we pass category, via "this.props.navigation.state.params.category".

This value is set from the HomeScreen onClick method. We save a firebase call.

File: screens/InventoryByCategoryScreen.js [hunk 3]
```diff
@@ -39,17 +44,37 @@ export default class InventoryByCategoryScreen extends React.Component {
...
-          {itemName} - Count: {itemCount}
-        </Text>
+        <TouchableOpacity 
+          key={itemName}
+          style={styles.touchable}>
+          <View style={styles.iconHolder}>
+            <Icon
+              name='food-apple-outline'
+              type='material-community'
+              size = {30} />
```
Comment (wayne-li2) [inline screens/InventoryByCategoryScreen.js]: Q: What is this doing? Do you have to do brackets around a number? :O

File: screens/InventoryByCategoryScreen.js [hunk 4]
```diff
@@ -39,17 +44,37 @@ export default class InventoryByCategoryScreen extends React.Component {
...
+          </View>
+          <View style={styles.iconHolder}>
+            <Text style={styles.itemName}> {itemName}</Text>
```
Comment (wayne-li2) [inline screens/InventoryByCategoryScreen.js]: Style: no space after ">"

File: screens/InventoryByCategoryScreen.js [hunk 5]
```diff
@@ -39,17 +44,37 @@ export default class InventoryByCategoryScreen extends React.Component {
...
+            <Text style={styles.itemCount}> {itemCount} in stock </Text>
```
Comment (wayne-li2) [inline screens/InventoryByCategoryScreen.js]: Style: No style after ">" and before "<"

File: screens/InventoryByCategoryScreen.js [hunk 6]
```diff
@@ -58,6 +83,11 @@ export default class InventoryByCategoryScreen extends React.Component {
   render() {
     return (
       <View style={styles.container}>
+        <View>
+          <Text style={styles.categoryName}>
+          {this.state.categoryDisplayName}
```
Comment (wayne-li2) [inline screens/InventoryByCategoryScreen.js]: Style: indent

File: screens/InventoryByCategoryScreen.js [hunk 7]
```diff
@@ -58,6 +83,11 @@ export default class InventoryByCategoryScreen extends React.Component {
   render() {
     return (
       <View style={styles.container}>
+        <View>
+          <Text style={styles.categoryName}>
+          {this.state.categoryDisplayName} 
+        </Text>
```
Comment (wayne-li2) [inline screens/InventoryByCategoryScreen.js]: Style: indent

**Comment by wayne-li2** [comment]
Pushed a commit -- just deleted some unused imports, and moved a style constant to the bottom of the page.

File: screens/InventoryByCategoryScreen.js (hunk 3 above)
Comment (samanthalai0914) [reply screens/InventoryByCategoryScreen.js]: yes lol

**Comment by samanthalai0914** [comment]
Just pushed that change you requested @wayne-li2

**Comment by wayne-li2** [comment]
Oh, this addresses much of #13

**Pull request merged**